tutorials/fiori-ios-scpms-starter-mission-04/fiori-ios-scpms-starter-mission-04.md

```diff
@@ -287,7 +289,7 @@ Because we already implemented a similar screen, the Product List, we can copy m
 
     ```
 
-    If you compile now, there shouldn't be any compile time errors. What you basically did is pasted a modified code copied from the `ProductsTableViewController.swift` class. You had to simply replace all the product definitions with customer and adjust the `tableView(_:cellForRowAt:)` method to use the product data similar to the Overview Screen.
+    If you compile now, there shouldn't be any compile time errors, if you get a compile time error for the `odataController` remember to make the `import SAPOfflineOData` import statement. What you basically did is pasted a modified code copied from the `ProductsTableViewController.swift` class. You had to simply replace all the product definitions with customer and adjust the `tableView(_:cellForRowAt:)` method to use the product data similar to the Overview Screen.
```

Comment:
"What you basically did is pasted a modified code copied"
Please correct grammar.